discord-groups/script.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+import { CANNOT_CONTAIN_GROUP, NO_SPACES_ALLOWED } from './constants.js';
+import {
+  getMembers,
+  getDiscordGroups,
+  addGroupRoleToMember,
+  createDiscordGroupRole,
+  getUserSelf,
+} from './utils.js';
+const groupTabs = document.querySelector('.groups-tab');
+const tabs = document.querySelectorAll('.groups-tab div');
+const sections = document.querySelectorAll('.manage-groups, .create-group');
+const loader = document.querySelector('.loader');
+const userIsNotVerifiedText = document.querySelector('.not-verified-tag');
+
+const membersIdNameObject = {};
+const membersData = await getMembers();
+
+// CREATED A MAP OF {id:username}
+membersData.forEach((member) => {
+  if (member.username) membersIdNameObject[member.id] = member.username;
+});
+
+/**
+ * GET SELF DATA
+ */
+const userSelfData = await getUserSelf();
+const userIsVerified = !!userSelfData.discordId;
+!userIsVerified && userIsNotVerifiedText.classList.remove('hidden');
+const memberAddRoleBody = {
+  userid: userSelfData?.discordId,
+  roleid: '',
+};
+
+/**
+ *
+ * FOR RENDERING GROUP ROLES IN 'MANAGE ROLES' TAB
+ */
+const groupsData = await getDiscordGroups();
```

Comment:
Suggestion: we don't need to call `getMembers();` `getDiscordGroups();` sequentially i.e., one after other, I think it would be better to call them parallely

Reply:
what do you mean by parallelly?